docs/en/manuals/html5.md

```diff
@@ -71,213 +64,157 @@ Internet Explorer 11 (Full screen)
 
+::: important
+The calculations for all scale modes include current screen DPI in case if you turn on `High Dpi` option in `game.project` (`Display` section)
+:::
 
-onSetProgress(current, max)
-: As the application is loaded, this callback receives the current progress. "current" contains the amount of data that has been loaded and "max" contains the total amount of data to load.
+### Downscale Fit and Fit
 
+For the `Fit` mode canvas size will be changed the way to show full game canvas on the screen with original proportions. The only difference in `Downscale Fit` is changing size only if the inner size of the webpage is smaller than the original canvas of the game, but doesn't scale-up when a webpage is bigger than the original game canvas.
 
-onDismissSplash()
-: This callback is invoked by the application when the application is loaded and it is about to begin execution and render to the canvas (i.e. the splash screen should be dismissed).
+![HTML5 Section](images/html5/html5_fit.png)
 
+### Stretch
 
-The default implementation also contains logic to deal with window resizing events. If you wish to change or replace the implementation, the main module provides two helper functions:
+For the `Stretch` mode canvas size will be changed to fully fill the inner size of the webpage.
 
-Module.matchToCanvas(id)
-: Accepts the id of a DOM element and adjusts its styled "width" and "height" properties to match the proportions of the canvas. The "marginTop" property is set to a value that centers the canvas vertically in the browser window.
+![HTML5 Section](images/html5/html5_stretch.png)
 
+### No Scale
+With `No Scale` mode the canvas size exactly the same as you predefined in `game.project` file, `[display]` section.
 
-Module.setMarginTop(id, sourcePixels)
-: Set the "marginTop" property of element "id" to a value "sourcePixels" that is expressed at a scale relative to the project settings application height. Internally, "sourcePixels" is multiplied by _scale_ before it's assigned to the top margin. The _scale_ is the actual browser window pixel height divided by the application pixel height as specified in the project settings.
+![HTML5 Section](images/html5/html5_no_scale.png)
 
+## Tokens
 
-::: important
-These methods are not available until the module script is loaded. See the default generated HTML file for example usage.
-:::
+We use [Mustache template language](https://mustache.github.io/mustache.5.html) for creation `index.html` file. When your are building the HTML and CSS files are passed through a compiler that is capable of replacing certain tokens with values that depend upon your project settings. These tokens are always encased in either double or triple curly braces (`{{TOKEN}}` or `{{{TOKEN}}}`), depending on whether character sequences should be escaped or not. This feature can be useful if you either make frequent changes to your project settings or intend for material to be reused in other projects.
```

Comment:
"for creation `index.html`" -> "for creation of the `index.html`"

"When your are building the HTML and CSS files" -> "When your are building or bundling, the HTML and CSS files"